src/main/scala/beam/agentsim/agents/ridehail/RideHailAgent.scala

```diff
@@ -510,30 +511,24 @@ class RideHailAgent(
       log.debug("state(RideHailingAgent.Offline): {}; Vehicle ID: {}", ev, vehicle.id)
       if (debugEnabled) outgoingMessages += ev
       handleNotifyVehicleResourceIdleReply(reply, data)
-    case ev @ Event(StartingRefuelSession(tick, _, triggerId), data) =>
+    case ev @ Event(StartingRefuelSession(tick, triggerId), _) =>
       // Due to parallelism window and dequeue process, tick could be unchronological
       val tickToUse = Math.max(tick, latestObservedTick)
       updateLatestObservedTick(tick)
       log.debug("state(RideHailAgent.Offline): {}; Vehicle ID: {}", ev, vehicle.id)
-      if (vehicle.isCAV) {
-        if (debugEnabled) outgoingMessages += ev
-        startRefueling(triggerId, Vector())
-        goto(Refueling)
-      } else {
-        holdTickAndTriggerId(tickToUse, triggerId)
```

Comment:
This is because refueling handles the parking spot no matter the vehicle type, right?